src/rpc/blockchain.cpp

```diff
@@ -125,35 +125,42 @@ struct MinterInfo {
     std::string OperatorAddress{};
     std::string RewardAddress{};
     uint64_t MintedBlocks{};
-    uint256 StakeModifier{};
+    std::string StakeModifier{};
```

Comment:
Sure we don't need the StakeModifier in it's raw form here for other use cases? 

Note: `MinterInfo` should be the source for everything where this stakeModifier is used, eventually. We shouldn't be having loose ad-hoc types all over the place. Please make sure that the common denominator is used here, and converted when it's used, rather than directly storing the converted types that's only needed on outer use cases like printing.

Reply:
Note: Merging, but marking for retrospective check to ensure the base types can absorb the most flexible use cases for future refactors.